### PR TITLE
Add timestamps as explicit elements of nvti.

### PR DESCRIPTION
**Pull request by janowagner**

### CODE REVIEW

File: base/nvti.c [hunk 1]
```diff
@@ -169,6 +169,9 @@ typedef struct nvti
   gchar *affected; /**< @brief Affected systems */
   gchar *impact;   /**< @brief Impact of vulnerability */
 
+  gint creation_time;     /**< @brief Time of creation, seconds since epoche */
```
Comment (mattmundell) [inline base/nvti.c]:
```suggestion
  gint creation_time;     /**< @brief Time of creation, seconds since epoch */
```

File: base/nvti.c [hunk 2]
```diff
@@ -169,6 +169,9 @@ typedef struct nvti
...
+  gint modification_time; /**< @brief Time of last change, sec. since epoche */
```
Comment (mattmundell) [inline base/nvti.c]:
```suggestion
  gint modification_time; /**< @brief Time of last change, sec. since epoch */
```

File: base/nvti.c [hunk 3]
```diff
@@ -464,6 +467,34 @@ nvti_impact (const nvti_t *n)
   return (n ? n->impact : NULL);
 }
 
+/**
+ * @brief Get the creation time.
+ *
+ * @param n The NVT Info structure of which the creation time should
+ *          be returned.
+ *
+ * @return The creation time in seconds since epoche.
```
Comment (mattmundell) [inline base/nvti.c]:
```suggestion
 * @return The creation time in seconds since epoch.
```

File: base/nvti.c [hunk 4]
```diff
@@ -464,6 +467,34 @@ nvti_impact (const nvti_t *n)
...
+  return (n ? n->creation_time : 0);
+}
+
+/**
+ * @brief Get the modification time.
+ *
+ * @param n The NVT Info structure of which the modification time should
+ *          be returned.
+ *
+ * @return The modification time in seconds since epoche.
```
Comment (mattmundell) [inline base/nvti.c]:
```suggestion
 * @return The modification time in seconds since epoch.
```

File: base/nvti.c (hunk 1 above)
Comment (mattmundell) [inline base/nvti.c]: I think these should be time_t instead of gint.  That's what we use in gvmd, and it matches the return from C's time().

Comment (janowagner) [reply base/nvti.c]: resolved.

File: base/nvti.c (hunk 2 above)
Comment (janowagner) [reply base/nvti.c]: Done.

File: base/nvti.c (hunk 3 above)
Comment (janowagner) [reply base/nvti.c]: Done.

File: base/nvti.c (hunk 4 above)
Comment (janowagner) [reply base/nvti.c]: Done.

File: base/nvti.c (hunk 1 above)
Comment (janowagner) [reply base/nvti.c]: Done.

**Pull request merged**